### ISSUE TITLE
Table overflow in Safari when open popup menu

### ISSUE THREAD

**Issue by ckissi** [tabler/tabler#148]
When I open popup menu in table there is an overflow in Safari (both x + y)
<img width="509" alt="table overflow" src="https://user-images.githubusercontent.com/5107590/40795426-5a30bd34-6502-11e8-8e5a-f059c4657473.png">

**Comment by dmarman** [comment]
Same in Chrome:

<img width="951" alt="screen shot 2018-06-12 at 23 14 27" src="https://user-images.githubusercontent.com/19510612/41317783-87d35a2e-6e96-11e8-93b8-52355e3e6c07.png">

**Comment by ckissi** [comment]
@dmarman in dashboard.css on line 2996 there is a 
```
.dropup,
.dropright,
.dropdown,
.dropleft {
  position: relative;
}
```
When I comment out this , it works. 
Let me know if it works for you.

**Comment by dmarman** [comment]
Worked like a charm! Did you find any side effect?

<img width="961" alt="screen shot 2018-06-14 at 00 33 20" src="https://user-images.githubusercontent.com/19510612/41382033-95e327e6-6f6a-11e8-91f2-e943b2bf286b.png">

**Comment by alkshendra** [comment]
> @dmarman in dashboard.css on line 2996 there is a
> 
> ```
> .dropup,
> .dropright,
> .dropdown,
> .dropleft {
>   position: relative;
> }
> ```
> When I comment out this , it works.
> Let me know if it works for you.

@ckissi This solution basically allows the dropdown to go out of the bounds of the popup toggle button.
It seems like there's no effect to the position of the dropdown cause there's an onScroll eventListener attached to the dropdown which, on scroll, fixes the dropdown position.

See the behavior in the below GIF:
![2018-10-02 23 49 19](https://user-images.githubusercontent.com/3149997/46368814-ceb68280-c69e-11e8-80f6-e99dca185d73.gif)

Notice how the position gets fixed on the scroll, and not on removing the `position: relative`.

**Comment by alkshendra** [comment]
TL;DR Fix: Remove the `table-responsive` wrapper from over the table element to fix this.

----
Longer Explanation:

This issue is happening cause the `<table>` element is wrapped into a wrapper with `table-responsive` class. The `table-responsive` has an `overflow-x` which basically makes tables horizontally scrollable.

However, this overflow interferes with the dropdown menus and does not let them go out of the wrapper.

Thus, it is advisable to not use the `table-responsive` wrapper over the table element when there's a dropdown inside the table.

The `table-responsive` wrapper makes the tables horizontally scorallable and should only be used when the table actually needs to be scrollable.

**Comment by jartaud** [comment]
@dmarman In my case, the only side effect is with the notification bell:
```
<a class="nav-link icon" @click="showNotifications" style="font-size:1.3em">
   <i class="fe fe-bell"></i>
   <span class="nav-unread" v-show="!liveNotifications.length || !unreadNofitications.length"></span>
</a>
```
`nav-unread` is being positioned at top right corner. My solution was to remove the span and add `.text-danger` to the bell.
